values/orange.yaml

```diff
@@ -28,3 +24,9 @@ packages: >-
     rsync
 
 kernel_package: linux-image-generic
+
+grub_packages: >-
```

Comment:
No need for a separate section here. You could have a package that extracts certains file from another, see for example https://github.com/rancher-sandbox/cOS-toolkit/blob/master/packages/cos/recovery-img/build.yaml. but  it can be as simple as having a package which `requires` the one that have the files you are interested in, and copy it back in the new